src/Serilog.Enrichers.CorrelationId/Serilog.Enrichers.CorrelationId.csproj

```diff
@@ -1,66 +1,29 @@
-    <DefineConstants>TRACE;DEBUG;FULLFRAMEWORK</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE;FULLFRAMEWORK</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10">
-      <HintPath>..\packages\Serilog.2.4.0\lib\net45\Serilog.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Web" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Accessors\IHttpContextAccessor.cs" />
-    <Compile Include="Accessors\IISPipelineHttpContextAccessor.cs" />
-    <Compile Include="..\Serilog.Enrichers.CorrelationId.Standard\Enrichers\CorrelationIdEnricher.cs" />
-    <Compile Include="..\Serilog.Enrichers.CorrelationId.Standard\Enrichers\CorrelationIdHeaderEnricher.cs" />
-    <Compile Include="..\Serilog.Enrichers.CorrelationId.Standard\CorrelationIdLoggerConfigurationExtensions.cs" />
-    <Compile Include="Extensions\NameValueCollectionExtensions.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <AssemblyName>Serilog.Enrichers.CorrelationId</AssemblyName>
+        <RootNamespace>Serilog</RootNamespace>
+        <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
+        <DefineConstants>NETCORE;NETSTANDARD;NETSTANDARD2_0</DefineConstants>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'net45'">
+        <DefineConstants>NET45;NETFULL</DefineConstants>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Serilog" Version="2.7.1" />
+    </ItemGroup>
+    
+    <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
+        <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.1" />
+    </ItemGroup>
+    
```

Comment:
you can add nuspec propes here, such as
```
<Target Name="SetNuspecProperties" BeforeTargets="GenerateNuspec">
    <PropertyGroup>
      <NuspecProperties>$(NuspecProperties);id=$(AssemblyName)</NuspecProperties>
      <NuspecProperties>$(NuspecProperties);title=$(AssemblyName)</NuspecProperties>
      <NuspecProperties>$(NuspecProperties);configuration=$(Configuration)</NuspecProperties>
      <NuspecProperties>$(NuspecProperties);version=$(PackageVersion)</NuspecProperties>
      <NuspecProperties>$(NuspecProperties);description=$(Description)</NuspecProperties>
      <NuspecProperties>$(NuspecProperties);tags=$(PackageTags)</NuspecProperties>
    </PropertyGroup>
    <Message Importance="high" Text="NuspecProperties: $(NuspecProperties)" />
  </Target>
```
it will be tacket from project descrption so on, 
and then you can use it 
it nuspec
```
 <metadata>
    <id>$id$</id>
    <version>$version$</version>
    <title>$title$</title>
    <owners>$author$</owners>
    <description>$description$</description>
    <tags>$tags$</tags>
</metadata>
```

Reply:
I think we can add this after we release. It seems more of a convenience than anything.